src/index.js

```diff
@@ -1,36 +1,76 @@
-const tasks = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Completed TO Do list project',
-    completed: true,
-    index: 2,
-  },
-];
+import './style.css';
 
+const taskInput = document.getElementById('input');
+const addButton = document.getElementById('button');
+const todoList = document.getElementById('todo-list');
+const completeButton = document.querySelector('.complete');
+
+// add task
 document.addEventListener('DOMContentLoaded', () => {
-  const todoList = document.getElementById('todo-list');
-  todoList.innerHTML = '';
-  todoList.style.listStyle = 'none';
+  const removeTask = (event) => {
+    const li = event.target.parentNode;
+    todoList.removeChild(li);
+  };
 
-  tasks.sort((a, b) => a.index - b.index);
+  const editTask = (event) => {
+    const li = event.target.parentNode;
+    const taskValue = li.firstChild.textContent.trim();
+    const newTaskValue = prompt('Edit task:', taskValue);
+    if (newTaskValue !== null && newTaskValue.trim() !== '') {
+      li.firstChild.textContent = newTaskValue.trim();
+    } else {
+      return;
+    }
```

Comment:
- [ ] Please fix your edit functionality it's not updating your data also please don't use the `prompt()` function because it does not look professional so kindly remove the `prompt` function and fix your edit function as well. :slightly_smiling_face: